app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,73 @@
+
+$explore-background: #449662;
+$box-background: #abe3c0;
+$box-footer-background: #4d785d;
+.home {
+  .left-column {
+    @include span-columns(10);
+    margin-top: 20px;
+  }
+  .right-column {
+    @include span-columns(2);
+    margin-top: 20px;
+
+    h2 {
+      @include padding(20px);
+    }
+
+    li {
+      @include padding(20px);
+      border-bottom: 1px solid #EBECE7;
```

Comment:
Color literals like `#EBECE7` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#EBECE7` should be written as `#ebece7`